db/migrate/20181031150129_rename_account_column_state_changed_at.rb

```diff
@@ -0,0 +1,5 @@
+class RenameAccountColumnStateChangedAt < ActiveRecord::Migration
+  def change
+    rename_column :accounts, :deleted_at, :state_changed_at
```

Comment:
So needs to have both columns combined in an index

Reply:
I don't understand what this means 😕